modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -137,10 +138,17 @@ class IIIFManifest extends StylePluginBase {
    */
   protected bool $structuredTextTermMemoized = FALSE;
 
+  /**
+   * Cached language code so we don't have to keep calling the lang service.
+   *
+   * @var string
+   */
+  protected $langCode = 'none';
```

Comment:
Do we need a default value here? 
```suggestion
  protected $langCode;
```

Reply:
I'm not sure. Is there ever a chance that `\Drupal::languageManager()->getCurrentLanguage()->getId()` ever returns a null or empty value? `none` is the IIIF Metadata spec's default value and I always hesitate to trust Drupal implicitly.

Reply:
Then again, I allow Drupal to simply overwrite it, so the point is moot.